frontend/test/resolution/testResolve.cpp

```diff
@@ -1739,6 +1739,83 @@ static void testInfiniteCycleBug() {
   std::ignore = resolveQualifiedTypeOfX(context, program1);
 }
 
+// a callable foraml (like a tuple) is preferred to functions in outer
```

Comment:
foraml->formal

Reply:
Thank you for your efforts to save me from Paul's spelling PRs :)